packages/@lightningjs/ui-components/src/mixins/withSelections/withSelections.mdx

```diff
@@ -58,11 +58,7 @@ class Example extends lng.Component {
 }
 ```
 
-<Canvas>
-  <Story id="utilities-withselections--with-selections" />
-</Canvas>
-
-Take a look at the preview above. The row of modified [Buttons](?path=/docs/components-button--button) has navigation managed by the [Row](?path=/docs/components-row--row) component, which is also wrapped in `withSelections`. Navigate with the arrow keys to see the `focus` state change for each [Button](?path=/docs/components-button--button). Hit the `Enter` key to set the currently focused `Button` to be `isSelected`, giving it an underline. Also notice that underline of the previously selected `Button` is removed.
+Take a look at the preview above. The row of modified [Buttons](?path=/docs/controls-button--button) has navigation managed by the [Row](?path=/docs/navigation-row--row) component, which is also wrapped in `withSelections`. Navigate with the arrow keys to see the `focus` state change for each [Button](?path=/docs/controls-button--button). Hit the `Enter` key to set the currently focused `Button` to be `isSelected`, giving it an underline. Also notice that underline of the previously selected `Button` is removed.
```

Comment:
I think this might be a bad merge, the top paragraph has the updates paths (component) vs navigation and controls

Reply:
I removed it. It's describing what was imported in the <Canvas> component, which we are no longer importing.